governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -0,0 +1,730 @@
+  )
+  const { rawConfig, dataIsLoading, connection } = usePythContext()
+  const { connected } = useWallet()
+  const [pythProgramClient, setPythProgramClient] =
+    useState<Program<PythOracle>>()
+
+  const openModal = () => {
+    setIsModalOpen(true)
+  }
+
+  const closeModal = () => {
+    setIsModalOpen(false)
+  }
+
+  const sortData = (data: any) => {
+    const sortedData: any = {}
+    Object.keys(data)
+      .sort()
+      .forEach((key) => {
+        const sortedInnerData: any = {}
+        Object.keys(data[key])
+          .sort()
+          .forEach((innerKey) => {
+            if (innerKey === 'metadata') {
+              sortedInnerData[innerKey] = sortObjectByKeys(data[key][innerKey])
+            } else if (innerKey === 'priceAccounts') {
+              // sort price accounts by address
+              sortedInnerData[innerKey] = data[key][innerKey].sort(
+                (priceAccount1: any, priceAccount2: any) =>
+                  priceAccount1.address.localeCompare(priceAccount2.address)
+              )
+              // sort price accounts keys
+              sortedInnerData[innerKey] = sortedInnerData[innerKey].map(
+                (priceAccount: any) => {
+                  const sortedPriceAccount: any = {}
+                  Object.keys(priceAccount)
+                    .sort()
+                    .forEach((priceAccountKey) => {
+                      if (priceAccountKey === 'publishers') {
+                        sortedPriceAccount[priceAccountKey] = priceAccount[
+                          priceAccountKey
+                        ].sort((pub1: string, pub2: string) =>
+                          pub1.localeCompare(pub2)
+                        )
+                      } else {
+                        sortedPriceAccount[priceAccountKey] =
+                          priceAccount[priceAccountKey]
+                      }
+                    })
+                  return sortedPriceAccount
+                }
+              )
+            } else {
+              sortedInnerData[innerKey] = data[key][innerKey]
+            }
+          })
+        sortedData[key] = sortedInnerData
+      })
+    return sortedData
+  }
+  const sortDataMemo = useCallback(sortData, [])
+
+  useEffect(() => {
+    if (!dataIsLoading && rawConfig && rawConfig.mappingAccounts.length > 0) {
+      const symbolToData: any = {}
+      rawConfig.mappingAccounts
+        .sort(
+          (mapping1, mapping2) =>
+            mapping2.products.length - mapping1.products.length
+        )[0]
+        .products.sort((product1, product2) =>
+          product1.metadata.symbol.localeCompare(product2.metadata.symbol)
+        )
+        .map((product) => {
+          symbolToData[product.metadata.symbol] = {
+            address: product.address.toBase58(),
+            metadata: {
+              ...product.metadata,
+            },
+            priceAccounts: product.priceAccounts.map((p) => ({
+              address: p.address.toBase58(),
```

Comment:
These address fields can be deleted imo for the same reason we delete `symbol` and `price_account`